diff --git a/Dave/Generated/CAN_NODE/can_node_conf.c b/Dave/Generated/CAN_NODE/can_node_conf.c
index ce1a2e7..4e2c3a2 100644
--- a/Dave/Generated/CAN_NODE/can_node_conf.c
+++ b/Dave/Generated/CAN_NODE/can_node_conf.c
@@ -93,6 +93,8 @@
  **********************************************************************************************************************/
 
 #include "can_node.h"
+#include "Can_Config.h"
+#include "Can_Config_XMC1400.h"
 
 
 
@@ -318,75 +320,11 @@ const CAN_NODE_t  CAN_NODE_0 = {
 
   .node_sr_ptr       = (CAN_NODE_SR_t*)&CAN_NODE_0_sr,
 
-  .lmobj_ptr         = { 
-
-      (CAN_NODE_LMO_t*)&CAN_NODE_0_LMO_01_Config, 
-
-      (CAN_NODE_LMO_t*)&CAN_NODE_0_LMO_02_Config,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL,  
-
-      NULL },
+  .lmobj_ptr         = CAN_MESSAGE_CONFIGS,
 
   .node_num          = 1U,
 
-  .mo_count          = 2U,
+  .mo_count          = NUMBER_OF_CAN_MESSAGE_CONFIGS,
 
   .loopback_enable         = false, 
 
